### DISCUSSION TITLE
Port of Micropython on Conexio Stratus Pro

### DISCUSSION THREAD

**Discussion by prenou** [Conexiotechnologies/conexio-firmware-sdk#2] [Q&A]
Hi,
Do you plan to port Micropython on the Conexio Stratus Pro, as it is sometimes easier for 1st level prototyping...
I saw that there are ports for boards with nRF9160:
- Nordic Semiconductor PCA10090
- Actinius Icarus
Thank you for feedback

**Comment by rajeev1986** [comment]
@prenou, any updates on the porting? We did provide the devices about a year ago.
